diff --git a/.config/emacs/init.el b/.config/emacs/init.el
index 022fc95..763da8f 100644
--- a/.config/emacs/init.el
+++ b/.config/emacs/init.el
@@ -20,12 +20,7 @@
 (custom-set-faces
  '(hl-line ((t (:background "gray20")))))
 
-;; check for graphical display and adjust visuals accordingly
-(when (display-graphic-p)
-  ;; invert only if the background color is the original one (e.g., "white")
-  (when (equal (face-background 'default) "white")
-    (invert-face 'default)) ; swap foreground and background colors
-  (setq frame-background-mode 'dark)) ; set frame background to dark
+(load-theme 'tango-dark)
 
 ;; set path for customise system
 (setq custom-file "~/.config/emacs/custom.el")
